### ISSUE TITLE
[v2.0.0-beta.1] Aliased fields are dropped from Action response

### ISSUE THREAD

**Issue by kyasu1** [hasura/graphql-engine#7056]
I have tested with the following official example repo.

[https://github.com/hasura/hasura-actions-examples/tree/master/auth](https://github.com/hasura/hasura-actions-examples/tree/master/auth)

By following the README, start the graphql-engine and nodejs server.

```
$ docker compose up -d
$ cd hasura
$ hasura migrate apply
$ hasura metadata apply
$ cd ../functions/nodejs-express
$ ENCRYPTION_KEY=SOMESECRETJWTKEY npm start
```

When I run the following mutation, `aliased_token` is dropped from the returned value.

```
mutation MyMutation {
  signup(name: "username", password: "somesecret", username: "hasura user") {
    id
    aliased_token: token
  }
}
```

The result I am getting on v2.0.0-beta.1 is:
```
{
  "data": {
    "signup": {
      "id": 9
    }
  }
}
```

Same on v1.2.0 is:
```
{
  "data": {
    "signup": {
      "id": 1,
      "aliased_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIiwibmFtZSI6InVzZXJuYW1lIiwiaWF0IjoxNjIzNjM5MDY5Ljk1MiwiaXNzIjoiaHR0cHM6Ly9teWFwcC5jb20vIiwiaHR0cHM6Ly9oYXN1cmEuaW8vand0L2NsYWltcyI6eyJ4LWhhc3VyYS1hbGxvd2VkLXJvbGVzIjpbInVzZXIiXSwieC1oYXN1cmEtdXNlci1pZCI6IjEiLCJ4LWhhc3VyYS1kZWZhdWx0LXJvbGUiOiJ1c2VyIiwieC1oYXN1cmEtcm9sZSI6InVzZXIifSwiZXhwIjoxNjIzNzI1NDY5fQ.fbiWdxfngdNb2eiDm-4LMk7MubcovCIqah5Uf_9gV1s"
    }
  }
}
```

**Comment by kolharsam** [comment]
@kyasu1 thank you for reporting this issue. 
I was able to reproduce the issue as per your instructions. A fix for the same will be out at the earliest!

**Comment by kolharsam** [comment]
fixed in c425b77
